src/main.rs

```diff
@@ -128,6 +130,11 @@ fn main() {
 }
 
 fn run_on_file_paths(file_path: &Path, options: &Options) {
+    if should_ignore_current_file(&options.ignored_files, file_path) {
+        log::debug!("file path {file_path:#?} found in ignored_files, will not sort");
+        return;
+    }
+
```

Comment:
Returning here early means that ignored_files will be ignored in each operation (write, check, print etc...)